pkg/reconciler/broker/broker.go

```diff
@@ -425,6 +433,9 @@ func (r *Reconciler) updateTargetsConfig(ctx context.Context) error {
 		if err != nil {
 			return fmt.Errorf("error creating targets ConfigMap: %w", err)
 		}
+		if err := r.reconcileDataPlaneDeployments(); err != nil {
+			r.Logger.Warnf("Error reconciling data plane deployments: %w", err)
```

Comment:
I'm not really sure if `%w` can work with formatter other than `fmt.Errorf`. Do you know?

Reply:
I think that's true. This should probably be `%v`.

Reply:
I thought I was doing Errorf... Good catch, thanks